x-pack/qa/rolling-upgrade-multi-cluster/build.gradle

```diff
@@ -0,0 +1,263 @@
+import org.elasticsearch.gradle.Version
+import org.elasticsearch.gradle.test.RestIntegTestTask
+
+apply plugin: 'elasticsearch.standalone-test'
+
+dependencies {
+    // "org.elasticsearch.plugin:x-pack-core:${version}" doesn't work with idea because the testArtifacts are also here
+    testCompile project(path: xpackModule('core'), configuration: 'default')
+    testCompile project(path: xpackModule('core'), configuration: 'testArtifacts') // to be moved in a later commit
+}
+
+// This is a top level task which we will add dependencies to below.
+// It is a single task that can be used to backcompat tests against all versions.
+task bwcTest {
+    description = 'Runs backwards compatibility tests.'
+    group = 'verification'
+}
+
+for (Version version : bwcVersions.wireCompatible) {
+    if (version.before('6.7.0')) {
```

Comment:
The main reason that I added this condition, is that 6.6.0 does not have restore from repository and I thought that therefor it wasn't worth testing. But I guess that was an overreaction, I will remove that condition and when backporting to other branches change it to: `if (version.before('6.5.x')) {`

Reply:
We have put some extra BWC logic in place so that 6.6 and 6.7 clusters can interact with each other (i.e. we use the old-style PUT FOLLOW INDEX logic on 6.7 as long as there are 6.6 nodes either in local cluster or remote cluster).